codespell_lib/data/dictionary.txt

```diff
@@ -16583,6 +16592,12 @@ intrumenting->instrumenting
 intruments->instruments
 intrusted->entrusted
 intstead->instead
+intstruct->instruct
```

Comment:
Definitely suggest `in struct` possibly suggest `int struct` if that can appear in some languages.

Reply:
→ 614ad8af3ef22c43bd64041dc2d0b1d2af3e99a4

Reply:
I didn't add `int struct` because in C, C++ and some other languages, `struct` is a keyword.